src/main/java/io/codurance/pouch/ResourceService.java

```diff
@@ -0,0 +1,23 @@
+package io.codurance.pouch;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class ResourceService {
```

Comment:
Do we really need this service?

Reply:
I was talking a bit about this in the update I've sent yesterday: To improve my previous solution of this, which left the Controller with too much responsibility, I paired with @andretorres-codurance and we came up with this service architecture. It enabled me to keep the Controller thin and clean, so it made sense to me.